docs/superpowers/plans/2026-07-30-lime-deep-links.md

```
@@ -0,0 +1,776 @@
+```bash
+git status --short
+git log --oneline origin/main..HEAD
+```
+
+`Apps/Shared/app_shared.yml` **must not** appear. A local `path:` override for it is parked in `git stash` (message: "local OTPKit path override (pre-deep-links)") and must stay out of the branch.
+
+- [ ] **Step 4: Record the manual verification checklist in the PR**
+
+This cannot be automated — the unit tests prove we build the URL we intend, not that Lime honours it. The PR body must carry:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)